### ISSUE TITLE
Item description bug?

### ISSUE THREAD

**Issue by Gawidev** [Tenome/Gurumin-Modding#5]
**Describe the bug**
Odd item description display bug going on. Seems like the text scrolling display is only scrolling the amount needed to show the first upgrade's specifications but not the others? On top of that the text of the last upgrade specs is layered on top of the rest of the text...
No clue if this is something caused by script updates or not, currently the only Item I have with a description that long, will follow up once I find another one so I can confirm if it happens with all items or just this one.

**Screenshots**
![Screenshot 2024-10-31 172641](https://github.com/user-attachments/assets/d255e555-fcbc-4037-a702-b10007337fb8)

https://github.com/user-attachments/assets/73673922-45ac-4aaf-9aa9-dd9d08a01fcb

**Save file**
https://mega.nz/#fm/dS83DZYb save 220 should have it visible.

**Comment by Tenome** [comment]
I think I've looked into this, and those item descriptions are stored in the .exe, and thus not something I can fix. I'll try the save on vanilla in a bit.

It's in vanilla too, so it's just some jank. I wonder if it's in the original JP release too.

**Comment by Gawidev** [comment]
Hmm, I guess it would technically be very easy to check if there was some cheat to unlock all items or something of the like. I can't seem to find info that might indicate in which versions this issue exists...
(I did manage to find a potential gurumin [info mine](http://falcom.xfu.jp/wiki/wiki.cgi/gurumin?page=%BE%AE%A5%CD%A5%BF%BD%B8), seems like its a jp community wiki made by some 2chan group way back? unsure of how useful the info there would ultimately be, worth eyeballing tho)
I'm going to mark the issue as "unfixable" and close it for now.

also going searching for old footage I found a lot of interesting stuff, like check out [this video](https://youtu.be/UehOivvEj20?feature=shared) and notice how different the game was in certain parts. Kinda makes me want to try out the original version to compare.
Oh and [the original trailer](https://youtu.be/vsR8_K4Hbvo?feature=shared) went really hard.
